### ISSUE TITLE
Detail pages

### ISSUE THREAD

**Issue by Jomula** [M-L-D-H/Closing-The-Gap-In-Non-Latin-Script-Data#353]
This is not important or urgent, but as I am following TEI conference, I saw a presentation on the DaSCH repository this morning. Most of it is not interesting for us, but it has a project browser with detail pages: https://meta.dasch.swiss/projects?_page=1&_limit=9

I am against further deepening the page hierarchy, but maybe we can think of a solution to provide a detail page for the projects in the frontend, especially when we gather more data that could be displayed (including download options and kinda-permalinks for the dataset)

**Comment by theodore-s-beers** [comment]
I've done a quick-and-dirty implementation of this. It can be a "semi-hidden feature" while we decide how it ought to work. You'll notice that, in a project card, you can now click the name, and it will take you to a detail page. See e.g.:

https://m-l-d-h.github.io/Closing-The-Gap-In-Non-Latin-Script-Data/project/#d1e6d69b-5e9a-4b4a-85ad-09aac56ed2d9

One question that I have is about the name of the new route. I went with `/project`, without really thinking about it, but now I wonder if `/entity` or `/entry` would be better, since we have the project vs. organization distinction in our database. My feeling at the moment is that I like `/entry`. @Jomula, what do you think?

**Comment by theodore-s-beers** [comment]
I've renamed the route to `/entry`. It can always be changed again.

**Comment by Jomula** [comment]
Looks good, I'd suggest to make it a little nicer with better typographic detachment between the value and the field names, maybe even using a monospace console font for the values? No specific idea, maybe we can play around a bit with the styling. But this would also be a nice place for the RDF graph information in a later stage.

**Comment by theodore-s-beers** [comment]
Yeah, the æsthetics can be improved a lot. I'll work on that—as long as the architecture makes sense to everyone.
